src/packages/ratmama/ratmama_parser.py

```diff
@@ -13,17 +13,20 @@
 
 import re
 from loguru import logger
-from typing import Optional, Dict
+from typing import Optional, Dict, TypedDict
 from src.config import CONFIG_MARKER
 from ..context import Context
 from ..rescue import Rescue
 from ..rules import rule
 from ..utils import Platforms
 
 
-_config: Dict = {}
+class _RatmammaConfig(TypedDict):
```

Comment:
[Error?] Are we going with `Ratmama` or `Ratmamma`? The file-name seems to indicate the former.

Reply:
... that wasn't even supposed to be comitted yet darnit.
will fix